contracts/contracts/coordination/Coordinator.sol

```diff
@@ -128,35 +131,57 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         __AccessControlDefaultAdminRules_init(0, _admin);
     }
 
+    function initializeNumberOfRituals() external reinitializer(2) {
```

Comment:
Should we include something to make sure that this function is called before the contract is used?

Reply:
I thought about that but honestly - it will only increase gas for checking and we can upgrade together with reinitialize (should be possible)

Reply:
Maybe let's add a dev comment reminding to use `upgradeAndCall` do it.

Reply:
✔️